src/Repository/WorklogRepository.php

```diff
@@ -114,23 +115,74 @@ public function findWorklogsByWorkerAndDateRange(string $workerIdentifier, \Date
 
     public function findBillableWorklogsByWorkerAndDateRange(string $workerIdentifier, \DateTime $dateFrom, \DateTime $dateTo)
     {
+        $nonBillableEpics = NonBillableEpicsEnum::getAsArray();
```

Comment:
These changes cannot work before issue.epics is a field and a migration has been added.
Add this to the Epics PR instead.

Reply:
I think that these changes belong here, as they are connected to the workload report and utilize the additions made in #183 . This PR is based off it as well.